config/aws-efs/manifests/stable/image-references

```diff
@@ -27,3 +27,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-kube-rbac-proxy:latest
+  - name: tools
```

Comment:
Hi @bertinatto the key tools needs update, we find build failed due to this reference, for tools it only can be `ose-tools `or `ose-network-tools`